week2/conf/bbuy_products.json

```diff
@@ -8,6 +8,14 @@
             "smarter_hyphens_filter",
             "lowercase"
           ]
+        },
+        "synonym": {
+          "tokenizer": "whitespace",
+          "filter": [
+            "lowercase",
```

Comment:
nice! i had forgotten to add this.

Di the query using synonymous ended up working for you?

On my side, I was getting the same results as without it, provided I had your same setup.

Reply:
Synonyms do work for me. Testing the analyzer was very helpful. For example
```
GET /bbuy_products/_analyze
{
  "analyzer": "synonym",
  "explain": "true",
  "text": ["xbox"]
}
```
produced words like playstation.
And querying for xbox in the main query produced more results with synonyms